src/login/LoginPage.jsx

```diff
@@ -91,6 +91,16 @@ class LoginPage extends React.Component {
     this.props.loginRequest(payload);
   }
 
+  handleOnFocus = (e) => {
+    const { errors } = this.state;
+    if (e.target.name === 'email') {
+      errors.emailOrUsername = '';
+    } else {
+      errors[e.target.name] = '';
+    }
```

Comment:
can't this be written as (we can change the email field name):
```suggestion
   errors[e.target.name] = '';
```